language/nl

```diff
@@ -146,7 +146,12 @@
 			"add": "Add Page",
 			"delete": "Delete Page",
 			"swap": "Change this Page"
+		},
+		"design-selector":{
```

Comment:
@RukayaDavids you need to start learning how to format your code, spacing and indenting. Code formatting helps the next developer understand what it is you wrote more easily. On this line can you please add a space after the colon `:` before the open brace `{`

Reply:
@RukayaDavids ??